net/wireguard/Makefile

```diff
@@ -9,10 +9,11 @@ include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=wireguard
 
-PKG_VERSION:=0.0.20160630
+PKG_VERSION:=0.0.20160708.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=WireGuard-experimental-$(PKG_VERSION).tar.xz
+PKG_SHA256SUM:=11faf795dd45ff0d13cdd204775b07e01fda596b4a9c2a1b326614c226e9725d
```

Comment:
It's confusing, but there is no `PKG_SHA256SUM`.  The build system currently is only aware of `PKG_MD5SUM`.  SHA256 checksum can be assigned to it and download script will know its type by checking the length (64 bytes instead of 32)